python/sdk/test/integration_test.py

```diff
@@ -611,3 +612,31 @@ def test_transformer_pytorch(integration_test_url, project_name, use_google_oaut
 
     # Undeploy other running model version endpoints
     undeploy_all_version()
+
+@pytest.mark.integration
+def test_standard_transformer_feast_pyfunc(integration_test_url, project_name, use_google_oauth):
+    merlin.set_url(integration_test_url, use_google_oauth=use_google_oauth)
+    merlin.set_project(project_name)
+    merlin.set_model("standard-transformer", ModelType.PYFUNC)
+
+    undeploy_all_version()
+    with merlin.new_model_version() as v:
+        v.log_pyfunc_model(model_instance=FeastModel(),
+                           conda_env="test/pyfunc/env.yaml",
+                           code_dir=["test"],
+                           artifacts={})
+
+    transformer_config_path = os.path.join("test/transformer", "standard_transformer_input.yaml")
+    transformer = StandardTransformer(config_file=transformer_config_path, enabled=False)
+
+    request_json = {"merchant_id": "1000"}
+    endpoint = merlin.deploy(v, transformer=transformer)
+    resp = requests.post(f"{endpoint.url}", json=request_json)
```

Comment:
Can you add assertion:
- check that the response contain `feast_features` json field
- check that the structure is correct (i.e. can be converted to pandas)
- ideally we can assert the completeness and correctness of entity and features returned